src/libs/ajax.js

```diff
@@ -444,6 +444,10 @@ const Groups = signal => ({
   }
 })
 
+const Resources = () => ({
+  leave: (samResourceType, samResourceId) => fetchSam(`api/resources/v2/${samResourceType}/${samResourceId}/leave`, _.merge(authOpts(), { method: 'DELETE' }))
+})
```

Comment:
Ajax has been moving towards having each top level piece in its own module. It would be nice to do that for this new piece.
https://github.com/DataBiosphere/terra-ui/tree/dev/src/libs/ajax

Reply:
love it. done in ea53e07898d9e7850e3403e36572d5a9984eb5b0